### DISCUSSION TITLE
Search bar in resources

### DISCUSSION THREAD

**Discussion by zeitgeistxx** [Resourcio-Community/Resourcio_Community-Website#172] [General]
Share your thoughts about implementation

**Comment by Abhinavcode13** [comment]
do I have to make a separate folder for search bar @Abhirup-02

**Comment by zeitgeistxx** [comment]
@Abhinavcode13 you can do it within resources.

**Comment by zeitgeistxx** [comment]
I was saying create a data file for all the resources, as object within a array and map over it to render the resources, and search implementation would be easy to in that way.
